File: mixedProblems/problem12.py
# Problem 12: How to check if a given year is a leap year

year = int(input("Enter Year: "))

if year%400==0:
    print(f"{year} is Leap Year.")
else:
    if year%4==0 and year%100!=0:
        print(f"{year} is Leap Year.")
    else:
        print(f"{year} is not Leap Year.")
    

File: mixedProblems/problem12.js
// Problem 12: How to check if a given year is a leap year
var year = 2012
if(year%400==0){
    console.log("The year is Leap year")
}else if(year%4==0 && year%100!=0){
    console.log("The year is Leap year")
}else{
    console.log("The year is not Leap year")
}